docs/control_center_docs/pulsing_module_getting_started.rst

```diff
@@ -0,0 +1,157 @@
+  For more on the *Direction* parameter and throttle mapping, see :ref:`throttle_direction`
+* **Max Velocity and Max Volts**: These parameters are available through the Tuning tab, and pair directly with the Mode parameter. This means that when your 
+  mode is set to velocity, your module will only ever map a throttle command as bounded by ± *Max Velocity*. When your mode is set to voltage, your module 
+  will only ever map a throttle as bounded by ± *Max Volts*. To learn more see :ref:`throttle_maximums`
+
+*******************************
+IFCI Configuration Parameters
+*******************************
+If you are controlling your pulsing module with :ref:`IFCI <controlling_ifci>`, you must configure your module's IFCI control value indices for 
+throttle, X, and Y commands. These parameters are available through the General tab as *Throttle Control Value Index*, *X Control Value Index*, and *Y Control Value Index*. 
+More information about these control values can be found at the link above.
+
+.. image:: ../_static/control_center_pics/pulsing_getting_started/unconfiged_cvis.png
+
+************************************
+Timeout Configuration
+************************************
+Vertiq's speed modules provide several configuration parameters regarding the module's timeout behavior. Timeouts are triggered after a 
+configured length of time of not receiving new commands. Timeouts and their configurations are covered in detail :ref:`here <manual_timeout>`.
+
+************************************
+Pulsing Torque Angle
+************************************
+This parameter is available through the General tab, and defines a correction factor for the difference between the torque vector and the torque induced by 
+the force vector. Please read :ref:`correcting_underactuated_torque` to learn more about properly setting this value for your vehicle.
+
+************************************
+Pulsing Frequency
+************************************
+This parameter is available through the General tab, and defines the number of pulses that happen per module rotation. This can be useful if the motor 
+is not directly driving the propeller.
+
+************************************
+Pulsing Voltage Mode
+************************************
+This parameter is available through the General tab, and determines how the module decides on its maximum pulsing voltage for scaling incoming pulsing commands. 
+In *Supply Voltage* mode, the supply voltage is used as the maximum pulsing voltage. In *Voltage Limit* mode, the maximum pulsing voltage is specified by :ref:`pulsing_voltage_limit`.
+
+.. _pulsing_voltage_limit:
+
+************************************
+Pulsing Voltage Limit
+************************************
+This parameter is available through the Tuning tab, and determines the maximum voltage to be applied to pulsing when *Pulsing Voltage Mode* is set to *Voltage Limit Mode*.
```

Comment:
Same as above, some useful info on this on the IFCI page I think, a more info link to there could be good: https://iqmotion.readthedocs.io/en/feature-add_pulsing_getting_started/manual/manual_ifci_control.html#iquart-flight-controller-interface